src/components/Hidden/__tests__/Hidden-test.js

```diff
@@ -47,23 +48,16 @@ it("calls hide when press Escape", () => {
 it("calls hide when click outside", () => {
   const props = {
     hide: jest.fn(),
-    hideonClickOutside: true
+    hideOnClickOutside: true,
+    visible: false
   };
+  const wrapper = mount(<Hidden {...props} />);
+  const clickEvent = new MouseEvent("click");
 
-  const wrapper = shallow(
```

Comment:
What is the (practical) difference between `shallow` and `mount` ? I'm curious about it.

Reply:
Shallow is real unit test (isolation, no children render) for this reason the tests were not going
Mount provides a full dom rendering, ideal for use cases where you have components that may interact with DOM APIs

Reply:
Thanks for explaining!